### ISSUE TITLE
Hack Doesn't Work

### ISSUE THREAD

**Issue by Acew0t** [iseekwonderful/csgoGlow#274]
I tried everything and it doesn't work.
I put it into terminal and this is what I get:

"Found CSGO PID: 609
Found the Client.dylib address: 0x11df97000 
Module should end at 0x11f49350c"

**Comment by Acew0t** [comment]
I am trying to just do a causal match and I did everything...
Nothing is working...

**Comment by ethano8225** [comment]
Did you update the offsets? A working build is forked on my profile. If you're on Mojave, make sure Terminal can control your computer. (System Preferences < Security and Privacy < Privacy < Accessibility and add Terminal to the list).

I haven't played in a while. Did you just update and it didn't work after the update?

**Comment by Acew0t** [comment]
No, It just doesn't work, do you have like discord so I can contact and we can do TeamViewer?

**Comment by Acew0t** [comment]
And I did add Terminal to the list.

**Comment by ethano8225** [comment]
I can't do a voice call right now, but if we do a discord call where you broadcast your screen I can type to you what you can try. (I'll see if it works on mine right now).

**Comment by Acew0t** [comment]
Great :D What is your discord?

**Comment by etan8225** [comment]
It is pullup#5386

**Comment by ethano8225** [comment]
I just tested my copy of it and it seems to work. Are you possible on Mojave?

**Comment by Acew0t** [comment]
I just checked your forked one and it works perfectly :D 
Thank you so much!
The other one didn't work for me :D

**Comment by amcgregor** [comment]
> (System Preferences < Security and Privacy < Privacy < Accessibility and add Terminal to the list).

Unless you want that malware copy of `ls` or `cp` or `ssh` or … to steal all your information (insert any standard command replaced by malware here), don't grant the whole Terminal process tree access to all of memory.  Just drag in the compiled `Wall` application: the one thing _actually_ being intentionally run that _actually_ needs these additional permissions.
